papers/oliver_beckstein/oliver_beckstein.rst

```diff
@@ -116,6 +116,40 @@ If you like to use MDAnalysis for your project please join our community_ board.
 .. _conda: https://anaconda.org/mdanalysis/dashboard
 .. _community: https://groups.google.com/forum/#!forum/mdnalysis-discussion
 
+Analysis Module
+---------------
+
+In the MDAnalysis.analysis module we provide a large variety of standard analysis algorithms, like RMSD, alignment :cite:`PuLiu_FastRMSD_2010`, native contacts :cite:`Best2013,Franklin2007`, as well as unique algorithms, like the LeaftleftFinder :cite:`Michaud-Agrawal:2011fu` and Path Similarty Analysis :cite:`Seyler:2015fk`.
+Historically the algorithms have been contributed by various researchers to satisfy their own needs, this lead to some fragmentation in the interface of different algorithms.
+We have recently started to unify the interface to the different algorithms with an `AnalysisBase` class.
+Currently PersistenceLength, InterRDF, LinearDensity and Contacts analysis have been ported.
+If applicable we also strive to make the API's to the algorithms generic.
+Most other tools hand the user analysis algorithms as black boxes, we want to avoid that and give the users all he needs to adapt an analysis to his/her needs.
```

Comment:
Most other tools hand the user analysis algorithms as black boxes. We ...

(Do not join two independent full clauses with a comma. Use periods or conjunctions.)